prep-javascript-objects/main.js

```diff
@@ -0,0 +1,14 @@
+var person = {
+  firstName: 'ignacio ',
+  lastName: 'dominguez',
+  hobby: 'gaming',
+  job: 'streamer',
+  previousJob: 'monogamist'
```

Comment:
rather than modifying the object here, we can update it and add properties later in our code.

```js
person.job = 'streamer';
```